en/spend-bitcoin.html

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
@JustAHappyKid At first glance it looks like you didn't add the url at the bottom of `en.yml`.